crates/store/re_types/src/blueprint/components/timeline_ext.rs

```diff
@@ -0,0 +1,15 @@
+use re_log_types::TimelineName;
+
+use super::Timeline;
+
+impl Timeline {
+    /// Return the name of the timeline.
+    pub fn timeline_name(&self) -> TimelineName {
+        TimelineName::from(self.0.as_str())
+    }
+
+    /// Set the name of the timeline.
+    pub fn set_timeline_name(&mut self, timeline_name: TimelineName) {
+        self.0 = timeline_name.as_str().into();
+    }
```

Comment:
Since `Timeline` only contains a name, and nothing else, I think it would be more natural to implement these as `impl From<re_log_types::TimelineName> for Timeline` and `impl From<Timeline> for re_log_types::TimelineName`